Assets/Scripts/Game/UserInterfaceWindows/DaggerfallTravelMapWindow.cs

```diff
@@ -1231,9 +1231,33 @@ bool FindLocation(string name)
             string[] locations = currentDFRegion.MapNames.OrderBy(p => p).ToArray();
             name = name.ToLower();
 
+            // this next code block addresses a bug reported by user "electrorobobody" on forums 
+            // the bug prevented one to find locations with their names being a substring of
+            // a different location, e.g. in province Daggerfall "Kingsly Manor" could not be
+            // found via search because there exists a "Buckingsly Manor" that was returned instead
```

Comment:
You can see the history of a single file from Github, so Lypyl can easily have access to your commit from [there](https://github.com/Nystul-the-Magician/daggerfall-unity/commits/2a205d958c4bcb0eadc07a37d4f5612e26968925/Assets/Scripts/Game/UserInterfaceWindows/DaggerfallTravelMapWindow.cs). Not saying that you should necessarily remove the comment, just pointing out this feature.